### PR TITLE
feat; document about reconnecting the closed database connection in Sequelize

### PR DESCRIPTION
**Pull request by fzn0x**
Closes sequelize/website#107

### CODE REVIEW

File: docs/getting-started.mdx [hunk 1]
```diff
@@ -121,6 +121,10 @@ try {
 Sequelize will keep the connection open by default, and use the same connection for all queries.
 If you need to close the connection, call `sequelize.close()` (which is asynchronous and returns a Promise).
 
+### Reconnecting the connection
+
+Sequelize can't reconnecting to closed connection, it's not reversible, but you can create a new instance of Sequelize to 'undo' the connection.
```
Comment (ephys) [inline docs/getting-started.mdx:126]: I think it's better suited as a note of the previous section than its own section

```suggestion
:::note

One `sequelize.close()` has been called, it's impossible to reopen a connection. You will need to create a new Sequelize instance to access your database again.

:::
```

Comment (fzn0x) [reply docs/getting-started.mdx:126]: > I think it's better suited as a note of the previous section than its own section

It is :D

File: docs/getting-started.mdx [hunk 2]
```diff
@@ -121,6 +121,12 @@ try {
 Sequelize will keep the connection open by default, and use the same connection for all queries.
 If you need to close the connection, call `sequelize.close()` (which is asynchronous and returns a Promise).
 
+:::note
+
+One `sequelize.close()` has been called, it's impossible to reopen a connection. You will need to create a new Sequelize instance to access your database again.
```
Comment (ephys) [inline docs/getting-started.mdx:126]: Sorry that's on me, I had a typo in my previous suggestion 🤦🏼‍♀️ 

```suggestion
Once `sequelize.close()` has been called, it's impossible to open a new connection. You will need to create a new Sequelize instance to access your database again.
```

**Review by ephys — CHANGES_REQUESTED** [review]

File: docs/getting-started.mdx (hunk 2 above)
Comment (fzn0x) [reply docs/getting-started.mdx]: Thanks

**Comment by ephys** [comment]
Ok nice :)
Can you add it to the v6 docs as well?

**Comment by fzn0x** [comment]
> Ok nice :) Can you add it to the v6 docs as well?



> Ok nice :) Can you add it to the v6 docs as well?

Sure, done adding it

**Review by ephys — APPROVED** [review]

**Pull request merged**